server/core/region_test.go

```diff
@@ -245,6 +245,36 @@ func (*testRegionKey) TestShouldRemoveFromSubTree(c *C) {
 	c.Assert(regions.shouldRemoveFromSubTree(region, origin), Equals, true)
 }
 
+func (*testRegionKey) TestGetStoreRegions(c *C) {
```

Comment:
Good job! But IMO, it's better to change the logic of TestRegion in region_test.go to test it.

Reply:
okay, I will look into it later, is it region_test.go inside region package or api package ?

Reply:
in tests/pdctl/region/region_test.go

Reply:
hi, I added the test case to ensure the learner shown (r3 in the test).
please check it out, thank you.